### ISSUE TITLE
`-g` cannot detect `.mk`

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#630]
(Original issue 923 created by leesei on 2013-11-07T07:36:24.990802+00:00)

`pygmentize -g Android.mk` incorrectly highlights the make file

`pygmentize -l make Android.mk` does a better job

**Comment by Anteru** [comment]
(Original issue 923 last updated on 2014-01-10T09:52:14.448353+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-09T18:04:09.182630+00:00)

which lexer does it recognize? can you attach the Android.mk file?

**Comment by Anteru** [comment]
(Original comment by leesei on 2014-01-10T04:27:04.826114+00:00)

I guess it's shell or no lexer, how to output the guessed lexer?

Here's the Android.mk I tested:
```makefile
# Copyright 2007 The Android Open Source Project
#
LOCAL_PATH:= $(call my-dir)
include $(CLEAR_VARS)

LOCAL_SRC_FILES := $(call all-subdir-java-files)
LOCAL_MODULE := pm
include $(BUILD_JAVA_LIBRARY)


include $(CLEAR_VARS)
ALL_PREBUILT += $(TARGET_OUT)/bin/pm
$(TARGET_OUT)/bin/pm : $(LOCAL_PATH)/pm | $(ACP)
        $(transform-prebuilt-to-target)
```

**Comment by Anteru** [comment]
(Original comment by leesei on 2014-01-10T04:32:22.871721+00:00)

Screenshots for pygmentize output

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-10T09:41:54.704682+00:00)

Closes #923: remove blanket nonzero analyse_text returns, and add an analyse_text for Makefile lexer

→ <<cset e8baab85fc0f>>

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-10T09:41:56.906125+00:00)

Thanks, that should now be working as intended.
